elastic/kibana
# [Uptime] Monitor management - fix breadcrumbs

## Summary

Fixes https://github.com/elastic/kibana/issues/121548
Fixes breadcrumbs for Monitor Management routes.

<img width="1771" alt="Screen Shot 2022-01-18 at 10 02 37 AM" src="https://user-images.githubusercontent.com/11356435/149962587-448f028c-0dfe-437c-8f10-6e47f59484da.png">
<img width="1776" alt="Screen Shot 2022-01-18 at 10 02 28 AM" src="https://user-images.githubusercontent.com/11356435/149962590-d4886208-068c-4612-87b7-d8a93b68b6d4.png">
<img width="1763" alt="Screen Shot 2022-01-18 at 10 02 20 AM" src="https://user-images.githubusercontent.com/11356435/149962592-e885b7d1-4550-4df2-bacf-d5722ac4c261.png">

### Testing
1. Add the following config to the `kibana.yml` file. For this test, you can use a local instance of ES: https://p.elstc.co/paste/63O1Z7Go#o1V84AgBIaU2CNNpbTWvhhSILnYMbF8BrORkxoN6Tfo
2. Start es with `yarn es snapshot` and kibana with `yarn kbn bootstrap & yarn start`
3. Navigate to Monitor Management table. Confirm the breadcrumbs are correct.
4. Click Add Monitor. Confirm the breadcrumbs are correct. Click the monitor management breadcrumb to ensure the href is correct.
5. Add a monitor. Navigate to the monitor management table. Click the edit button. Ensure that the breadcrumbs are correct. Click the monitor management breadcrumb to ensure the href is correct.
